.eslintrc

```diff
@@ -488,6 +488,31 @@
         ]
       }
     },
+    {
```

Comment:
Should we add a similar rule to force using `@wordpress/element` instead of `react` in places like the Web Stories block? Just for consistency

Reply:
I will do this in a follow up PR.

Reply:
Whatever works best for you.

But we should add WP-specific packages like activation-notice, stories-block, etc. to `excludedFiles` here in this PR.